src/main/java/org/influxdb/impl/InfluxDBService.java

```diff
@@ -39,49 +39,44 @@
    */
   @POST("write")
-  public Call<ResponseBody> writePoints(@Query(U) String username,
-      @Query(P) String password, @Query(DB) String database,
+  public Call<ResponseBody> writePoints(@Query(DB) String database,
       @Query(RP) String retentionPolicy, @Query(PRECISION) String precision,
       @Query(CONSISTENCY) String consistency, @Body RequestBody batchPoints);
 
   @GET("query")
-  public Call<QueryResult> query(@Query(U) String username, @Query(P) String password, @Query(DB) String db,
+  public Call<QueryResult> query(@Query(DB) String db,
       @Query(EPOCH) String epoch, @Query(value = Q, encoded = true) String query);
 
   @POST("query")
-  public Call<QueryResult> query(@Query(U) String username, @Query(P) String password, @Query(DB) String db,
+  public Call<QueryResult> query(@Query(DB) String db,
           @Query(EPOCH) String epoch, @Query(value = Q, encoded = true) String query,
           @Query(value = PARAMS, encoded = true) String params);
 
   @GET("query")
-  public Call<QueryResult> query(@Query(U) String username, @Query(P) String password, @Query(DB) String db,
+  public Call<QueryResult> query(@Query(DB) String db,
       @Query(value = Q, encoded = true) String query);
 
   @POST("query")
-  public Call<QueryResult> postQuery(@Query(U) String username, @Query(P) String password, @Query(DB) String db,
+  public Call<QueryResult> postQuery(@Query(DB) String db,
       @Query(value = Q, encoded = true) String query);
 
   @POST("query")
-  public Call<QueryResult> postQuery(@Query(U) String username, @Query(P) String password, @Query(DB) String db,
+  public Call<QueryResult> postQuery(@Query(DB) String db,
           @Query(value = Q, encoded = true) String query, @Query(value = PARAMS, encoded = true) String params);
 
   @GET("query")
-  public Call<QueryResult> query(@Query(U) String username, @Query(P) String password,
-      @Query(value = Q, encoded = true) String query);
+  public Call<QueryResult> query(@Query(value = Q, encoded = true) String query);
 
   @POST("query")
-  public Call<QueryResult> postQuery(@Query(U) String username,
-      @Query(P) String password, @Query(value = Q, encoded = true) String query);
+  public Call<QueryResult> postQuery(@Query(value = Q, encoded = true) String query);
 
   @Streaming
   @GET("query?chunked=true")
-  public Call<ResponseBody> query(@Query(U) String username,
-      @Query(P) String password, @Query(DB) String db, @Query(value = Q, encoded = true) String query,
+  public Call<ResponseBody> query(@Query(DB) String db, @Query(value = Q, encoded = true) String query,
       @Query(CHUNK_SIZE) int chunkSize);
 
   @Streaming
   @POST("query?chunked=true")
-  public Call<ResponseBody> query(@Query(U) String username,
-          @Query(P) String password, @Query(DB) String db, @Query(value = Q, encoded = true) String query,
+  public Call<ResponseBody> query(@Query(DB) String db, @Query(value = Q, encoded = true) String query,
```

Comment:
parameters removed: (_non-backward compatible change_)

Reply:
No, InfluxDBService provides only a internal API which @lxhoan adopted, the only visible API is InfluxDB,java and that didnt change.

Or did i miss something ?

Reply:
the method signatures is public however the interface (InfluxDBService) itself is visible to org.influxdb.impl only, so I don't think user can import this interface in their implementation